api/graphql/reservations/reservation_serializers.py

```diff
@@ -131,6 +141,18 @@ def validate(self, data):
 
         return data
 
+    def _get_biggest_buffer_time_from_reservation_units(
+        self, field: str, reservation_units: List[ReservationUnit]
+    ) -> [datetime.timedelta]:
+        buffer_times = [
+            getattr(res_unit, field)
+            for res_unit in reservation_units
+            if getattr(res_unit, field, None) is not None
+        ]
+        if len(buffer_times) > 0:
+            return max(buffer_times)
+        return None
```

Comment:
Minor thing, but I think we could simplify this to:
```suggestion
        return max(buffer_times, default=None)
```